src/styles/main.scss

```
@@ -1,3 +1,67 @@
+$baza: 40px;
+
 body {
   margin: 0;
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 730px;
+  
+  & :first-child {
+    margin-left: 622px;
+  }
+
+  @for $index from 1 through 31 {
+    & :nth-child(#{$index})::before {
+      position: absolute;
+      content: "#{$index}";
+      width: 20px ;
+      height: 20px;
+      left: $baza;
+      top: $baza;
+      font-family: Arial, sans-serif;
+      font-size: 22px;
+      color: black;
+    }
+  }
+
+  &__cell {
+    position: relative;
+    width: 100px;
+    height: 100px;
+    margin: 1px;
+    border: 1px solid black;
+    background-color: #eee;
+    cursor: pointer;
+    transition: 0.5s;
+ 
+    &:hover {
+      background-color: pink;
+    }
+
+    &:active {
+      transition: 0.5s;
+      transform: translateY(-20px);
+    }
+  }
+  
+  @for $i from 1 through 3 {
+    &__cell--month-length--#{(28+$i)} {
+      display: none;
+    }
+  }
```

this style won't work as expected in the task. the class `calendar--month-length--29` should show only 29 days. it means you hide all children with an index of more than 29.
it should look something like this:
```suggestion
  @for $i from 28 through 31 {
    &--month-length--#{($i)} :nth-child(n + #{$i+1}) {
      display: none;
    }
  }
```